agendas/2025/03-Mar/13-community-working-group-march-2025.md

```diff
@@ -49,7 +50,8 @@
 1. Check for
    [ready for review agenda items](https://github.com/graphql/community-wg/issues?q=is%3Aissue+is%3Aopen+label%3A%22Ready+for+review+%F0%9F%99%8C%22+sort%3Aupdated-desc)
    (5m, Host)
-1. Make the Discord channel public? (5m, All)
-2. How to involve the community in GraphQL conf organization? (15m, Martin, All)
+1. [Align on main goals for the working group]([https://github.com/graphql/community-wg/issues/28](https://github.com/graphql/community-wg/issues/28#issuecomment-2718221158)) (15m, Urigo)
+1. [Align on how to manage intiatives (ideas, actions, champions, stages and tracking progress)]([https://github.com/graphql/community-wg/issues/28](https://github.com/graphql/community-wg/issues/28#issuecomment-2718221158)) (10m, Urigo)
+1. [Making progress on the website redesign](https://github.com/graphql/community-wg/issues/21) (5m, Urigo)
+1. How to involve the community in GraphQL conf organization? (15m, Martin, All)
   1. Where is help needed?
```

Comment:
[Quoting @doc-jones](https://github.com/graphql/community-wg/issues/25#issuecomment-2714847394) , "Volunteers are more comfortable contributing if they know how to get started and what is being asked.". In light of this, suggestion to merge those 2 items into one:

```suggestion
1. Where is help needed? (15m, All)
  1. https://github.com/graphql/community-wg/issues/25#issuecomment-2714847394
  1. https://github.com/graphql/community-wg/issues/26
```